project/GenerateAnyVals.scala

```diff
@@ -164,8 +166,8 @@ import scala.language.implicitConversions"""
       val xs1 = List(mkCoercions, mkUnaryOps, mkStringOps, mkShiftOps) map (xs => if (xs.isEmpty) xs else xs :+ "")
       val xs2 = List(
         mkBinOpsGroup(comparisonOps, numeric, _ => Z),
-        mkBinOpsGroup(bitwiseOps, cardinal, this opType _),
-        mkBinOpsGroup(otherOps, numeric, this opType _)
+        mkBinOpsGroup(bitwiseOps, cardinal, this opType),
+        mkBinOpsGroup(otherOps, numeric, this opType)
```

Comment:
not going to look at why `opType` doesn't suffice, but saying `this f arg` was a style quirk.

Reply:
Reverted.